scheduler/src/cook/config.clj

```diff
@@ -199,9 +199,11 @@
                                      ((util/lazy-load-var 'cook.impersonation/create-impersonation-middleware) impersonators)
                                      {:json-value "config-impersonation"})))
      :rate-limit (fnk [[:config {rate-limit nil}]]
-                   (let [{:keys [user-limit-per-m]
+                   (let [{:keys [user-limit-per-m rate-limit-expire-min]
                           :or {user-limit-per-m 600}} rate-limit]
-                     {:user-limit (->UserRateLimit :user-limit user-limit-per-m (t/minutes 1))}))
+                     {:user-limit (->UserRateLimit :user-limit user-limit-per-m (t/minutes 1))
+                      :job-creation (:job-creation rate-limit)
+                      :rate-limit-expire-min (or rate-limit-expire-min 120)}))
```

Comment:
We can default `rate-limit-expire-min` in the `or` clause above like for `user-limit-per-m`

Reply:
Nitpick: lexicographic order